ci-operator/step-registry/openstack/conf/createfips/openstack-conf-createfips-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml
+OS_CLOUD=$(<"${SHARED_DIR}"/OS_CLOUD)
+export OS_CLOUD
```

Comment:
Do we need this line (OS_CLOUD should be set by workflow)

Reply:
Not strictly needed but we should keep it defined in ${SHARED_DIR} for concistency